packages/block-library/src/preformatted/style.scss

```diff
@@ -0,0 +1,4 @@
+.wp-block-preformatted {
+	font-size: var(--wp--preset--font-size--extra-small, 1em);
```

Comment:
:wave: We should remove the CSS Custom Property from here. I've seen that the code block recently got it added as well in https://github.com/WordPress/gutenberg/pull/27294/files#r540890153 and I've prepared a PR that fixes and explain why we shouldn't do that https://github.com/WordPress/gutenberg/pull/27672